RecoLocalCalo/EcalRecProducers/python/ecalUncalibRecHitPhase2_cff.py

```diff
@@ -1,4 +1,52 @@
 import FWCore.ParameterSet.Config as cms
+from HeterogeneousCore.CUDACore.SwitchProducerCUDA import SwitchProducerCUDA
+from Configuration.ProcessModifiers.gpu_cff import gpu
 
-from RecoLocalCalo.EcalRecProducers.ecalUncalibRecHitPhase2_cfi import *
+
+from RecoLocalCalo.EcalRecProducers.ecalUncalibRecHitPhase2_cfi import ecalUncalibRecHitPhase2 as _ecalUncalibRecHitPhase2
+ecalUncalibRecHitPhase2 = SwitchProducerCUDA(
+  cpu = _ecalUncalibRecHitPhase2.clone()
+)
+
+# cpu weights
 ecalUncalibRecHitPhase2Task = cms.Task(ecalUncalibRecHitPhase2)
+
+# conditions used on gpu
+
+
+from RecoLocalCalo.EcalRecProducers.ecalPh2DigiToGPUProducer_cfi import ecalPh2DigiToGPUProducer as _ecalPh2DigiToGPUProducer
+ecalPh2DigiToGPUProducer = _ecalPh2DigiToGPUProducer.clone()
+
+# gpu weights
+from RecoLocalCalo.EcalRecProducers.ecalUncalibRecHitPhase2GPU_cfi import ecalUncalibRecHitPhase2GPU as _ecalUncalibRecHitPhase2GPU
+ecalUncalibRecHitPhase2GPU = _ecalUncalibRecHitPhase2GPU.clone(
+  digisLabelEB = cms.InputTag('ecalPh2DigiToGPUProducer', 'ebDigis')
```

Comment:
It would be better to avoid explicit types in `clone()` when the cloned-from object already has the parameter
```suggestion
  digisLabelEB = ('ecalPh2DigiToGPUProducer', 'ebDigis')
```